content/docs/next-release/spm.md

```diff
@@ -157,6 +157,10 @@ The following configuration is required to enable the SPM feature:
   - Optional: Set `--prometheus.server-url` (or `PROMETHEUS_SERVER_URL` environment variable)
     to the URL of the prometheus server. Default: http://localhost:9090.
 
+{{< info >}}
+[PostgreSQL with Promscale](https://github.com/timescale/promscale#promscale-for-jaeger-and-opentelemetry) shall be used as an single storage backend for both [Prometheus metrics](https://docs.timescale.com/promscale/latest/send-data/prometheus/) and [Jaeger traces](https://docs.timescale.com/promscale/latest/send-data/jaeger/).
+{{< /info >}}
+
```

Comment:
I think this is too aggressive. It feels like pushing Promscale too much in their docs. I don't think it makes sense and I would keep out of this PR. We could consider doing it in the future.